schema_editor/app/scripts/views/settings/list-controller.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+    'use strict';
+
+    /**
+     * @ngInject
+     */
+    function SettingsListController ($log, $scope, BlackSpotConfig, Notifications, Utils) {
+
+        var ctl = this;
+        ctl.blackSpotSeverityUpdateClicked = blackSpotSeverityUpdateClicked;
+        initialize();
+
+        function initialize() {
+            BlackSpotConfig.query().$promise.then(function (results) {
+                // The BlackSpotConfig setting is a singleton -- there is guaranteed to be only one
+                ctl.blackSpotConfig = results[0];
+            });
+        }
+
+        function blackSpotSeverityUpdateClicked() {
+            /* jshint camelcase: false */
+            var threshold = ctl.blackSpotConfig.severity_percentile_threshold;
+            /* jshint camelcase: true */
+
+            // Validate the black spot severity threshold value.
+            // The threshold will be undefined if it didn't meet the criteria of the input field.
+            // The range check has been added as well, just in case, but probably isn't needed.
+            if (threshold === undefined || threshold < 0 || threshold > 1) {
```

Comment:
There's server-side validation for this as well but it's good to have it client-side too so that users receive quicker feedback.